content/pages/education/gi-bill/post-9-11/ch-33-benefit-backup.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Check Benefit
+entryname: post-911-gib-status
+layout: page-react.html
+includeBreadcrumbs: true
+in_maintenance: false
```

Comment:
We might just be able to toggle this guy instead of changing the content

Reply:
:man_facepalming: